doc/analyzerDoc.md

```diff
@@ -0,0 +1,54 @@
+## ApproxCountDistinct
+Compute approximated count distinct with [HyperLogLogPlusPlus](https://en.wikipedia.org/wiki/HyperLogLog#HLL++)
+## ApproxQuantile
+Approximate quantile analyzer. 
+Yields the x-quantile; a value, for which it holds true, that exaclty x percent of the total values are smaller than the yielded value.
+The quantile x can be passed to the analyzer as parameter, x has to be a number between 0 and 1. Choosing 0.5 will yield the median.
+The allowed relative error compared to the exact quantile can be configured with `relativeError` parameter. A `relativeError = 0.0` would yield the exact quantile while increasing the computational load.
+## ApproxQuantiles
+The same as ApproxQuantile, can be given a sequence of quantiles.
+## Completeness
+Completeness is the fraction of the number of non-null values divided by the number of all values in a column.
+## Compliance
+Compliance is a measure of the fraction of rows that complies with the given constraint. This constraint may span multiple columns.
+e.g Given the constraint is "att1 > 3" and the data frame has 5 rows with an att1 column value greater than
+3 and 10 rows with a value lower than 3; a DoubleMetric of 0.33 would be returned.
+## Correlation
+Computes the [pearson correlation coefficient](https://en.wikipedia.org/wiki/Pearson_correlation_coefficient) between the two given columns.
+It returns a value between -1 and +1, where +1 is total positive linear correlation, 0 is no linear correlation, and −1 is total negative linear correlation.
+## CountDistinct
+Number of distinct values in the column.
+## DataType
+Yields a distribution map, including the overall number of values of each datatype and the percentage of each datatype. 
+Possible datatypes are Boolean, Fractional, Integral and String. If the type can't be determined, Unknown is returned.
+## Distinctness
+Distinctness is the fraction of the number of distinct values devided by the number of all values of a column. Can be given a sequence of columns.
+## Entropy
+[Entropy](https://en.wikipedia.org/wiki/Entropy_(information_theory)) is a measure of the level of information contained in a message. Given the probability distribution over values in a column, it describes how many bits are required to identify a value. A highly diverse column will result in high entropy. Few different values will result in low entropy.
+## Histogram
+Divides the columns of the values in the given column into buckets. The number of buckets has to be given by the user and has to be smaller than or euqal to 1000. 
```

Comment:
This is still not a precise description in my opinion. Choosing the number of buckets will be understood as specifying into how many buckets a continuous value range should be divided. What actually happens here is that values with low cardinality are filtered out and simply not included in the result. This is a totally different behaviour and it should be explicitly stated in the description.
Aditionally I find the first sentence confusing. The analyzer doesn't create buckets as far as I know. It simply counts the frequency of all occuring values within a column, even if they are numeric.

Reply:
Fixed it